homeassistant/components/climate/ecobee.py

```diff
@@ -158,16 +159,12 @@ def current_temperature(self):
     @property
     def target_temperature_low(self):
         """Return the lower bound temperature we try to reach."""
-        if self.current_operation == STATE_AUTO:
-            return self.thermostat['runtime']['desiredHeat'] / 10.0
-        return None
+        return self.thermostat['runtime']['desiredHeat'] / 10.0
```

Comment:
We shouldn't do this, although I guess we should filter this out at the climate level. There is no upper/lower bound when we just have a target temp.